recipes/openpam/all/test_package/test_package.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <security/pam_types.h>
+#include <security/openpam.h>
+#include <security/pam_constants.h>
+
+
+int main(void) {
+
+    int x = PAM_SUCCESS;
+
+    return EXIT_SUCCESS;
+}
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>
#include <security/pam_types.h>
#include <security/openpam.h>
#include <security/pam_constants.h>
#include <security/pam_appl.h>


int main(void) {
    pam_handle_t* pamh;
    struct pam_conv pamc;
    const char *user;

    pam_start("yes", user, &pamc, &pamh);
    pam_end(pamh, PAM_SUCCESS);

    return EXIT_SUCCESS;
}

```
Need to consume some function from the library to check runtime linkage.